Config.uk

```diff
@@ -235,3 +235,10 @@ config UK_NAME
 	default UK_DEFNAME
 	help
 	  Name to be used for final image
+
+# The following are selected by libraries that provide options to bypass security.
+config CONFIG_UK_TAINT_INSECURE_CSPRNG
```

Comment:
Sounds like something that would have the `HAVE_*` semantics? :thinking:

Reply:
Yes for sure, that could also be an option. I used `TAINT` here instead to group future options that taint Unikraft.

Reply:
Ok I will approve anyway. I am fine either way with this.